app/controllers/users.js

```diff
@@ -0,0 +1,14 @@
+const { signup } = require('../services/users');
+const { info, error } = require('../logger');
+
+exports.signup = async (req, res, next) => {
+  try {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Don't disable eslint rules please, it is better do use a serializer to generate the desired response. You can create a folder named serializers a create a function to generate the response for the siugnup.